packages/eslint-plugin-svelte/tests/fixtures/rules/prefer-svelte-reactivity/invalid/arrow-function-return01-input.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+	const fn = () => {
+		const variable = new Set([1, 2, 1, 3, 3]);
+		return variable;
+	}
+</script>
+
+{fn().has(42)}
```

Comment:
Hmm. This is actually fine as is, right? (It shouldn't be reported, right?)
Could you change it to track the value the function returns, check if there's a mutation, and only report that?

Reply:
I've been thinking about this more and you are right. I think this needs a change of approach. I will probably make an alternative PR.